tasks/F3Parameters.ts

```diff
@@ -3,12 +3,24 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
 import fs from "fs";
 import zlib from "zlib";
 
-task("transferOwnership", "Transfers ownership of the contract to a new account")
+task("queryOwnership", "Queries the current owner of the contract")
   .addParam("contract", "The address of the contract")
+  .setAction(async (taskArgs: { contractAddress: string }, hre: HardhatRuntimeEnvironment) => {
+    const contractAddress = taskArgs.contractAddress;
+
+    const F3Parameters = await hre.ethers.getContractFactory("F3Parameters");
+    const contract = F3Parameters.attach(contractAddress);
+
+    const ownerAddress = await contract.owner();
+    console.log(`The current owner of the contract at ${contractAddress} is ${ownerAddress}`);
+  });
+
+task("transferOwnership", "Transfers ownership of the contract to a new account")
+  .addParam("contractAddress", "The address of the contract")
```

Comment:
Throughout changed to contractAddress to be self documenting.

Reply:
This was coming from when I was looking at the code in the task without looking at the param text above.  The param text makes it clear, so I could remove this.

Reply:
The task params have to be lower case only, and IMO shorter the better, the CLI lines are multiple lines in terminal without extra verbosity.

Reply:
Ack sounds good.  I'll close this given https://github.com/filecoin-project/f3-activation-contract/pull/8